nodes/test/tool_git/test_tools.py

```
@@ -0,0 +1,508 @@
+# Helpers
+# ---------------------------------------------------------------------------
+
+
+def _make_instance() -> IInstance:
+    inst = IInstance.__new__(IInstance)
+    inst.IGlobal = MagicMock()
+    inst.IGlobal.repo = MagicMock(spec=GitRepo)
+    return inst
+
+
+def _invoke(inst: IInstance, tool_name: str, args: Dict[str, Any] = {}) -> str:
```

Ruff (0.15.12)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)